app/(home)/index.tsx

```diff
@@ -400,19 +365,41 @@ export default function Home() {
             ) : (
               <NoGoalSet type={"People"} />
             )}
+
+            <View
+              style={{
+                paddingHorizontal: 10,
+              }}
+            >
+              <Text
+                style={{
+                  fontSize: 16,
+                  fontFamily: "Rubik_500Medium",
+                }}
+              >
+                Featured
+              </Text>
+            </View>
+            <View>
+              <FeaturedCard />
```

Comment:
Would be nice if you simulated loading these from a server, maybe use setTimeout to simulate and handle loading